### PR TITLE
Use split-lines

### PR DESCRIPTION
**Pull request by timdp**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 'use strict';
+
```
Comment (SamVerschueren) [inline index.js]: Remove empty line

Comment (timdp) [inline index.js]: Both or just this one? I already get criticized for not adding enough blank lines. :stuck_out_tongue:

Comment (SamVerschueren) [inline index.js]: Not sure about the other one, that's up to Sindre :p

**Comment by sindresorhus** [comment]
Thanks @timdp :)

**Comment by timdp** [comment]
yw :-)

**Comment by dgoguerra** [comment]
Fair question, why is it preferable to add that dependency? split-lines just does the same

**Comment by sindresorhus** [comment]
@dgoguerra Read https://github.com/sindresorhus/ama/issues/10#issuecomment-117766328 and https://github.com/sindresorhus/ama/issues/367#issuecomment-204355235.

Sorry for locking this, but I don't have energy to discuss.

**Pull request merged**